index.ts

```diff
@@ -77,9 +75,10 @@ export function status() {
  */
 
 // Membrane comes with several built-in utils that your programs can install, like `sms` and `email`.
-// Your workspace has `sms` and `email` installed and this program includes them as dependencies. We'll cover how to add new dependencies later.
+// Your workspace has `sms` and `email` installed, and this program includes them as dependencies.
```

Comment:
Should we use "connections" here?

Reply:
We could also rename `DEPENDENCIES` in the CONFIG panel